### ISSUE TITLE
Modul Counter Mannschaftsstatistik

### ISSUE THREAD

**Issue by diddipoeler** [diddipoeler/sportsmanagement#504]
Ich migriere gerade unsere Seite von Joomla 2.5 und JL2 zu Joomla 3.5 und JSM. Ich habe dann ein Modul mit einem Counter wie hier ( bootsnipp.com/snippets/97mWl ) eingebaut. Schön wäre das evtl. auch mit Werten aus der Mannschaftsstatistik. Zeige mit die Tore / Spiele ect. der Mannschaft XYZ in den Projekten XYZ. Dann wird der Gesamtwert in einem solchen Counter hochgezählt. Wenn Ihr ein solches Modul realisiert, würde ich mich auch um das Frontend-Layout in Bootstrap kümmern sofern ich das hin bekomme.

**Comment by tdittmann** [comment]
Ein erster Entwurf:
![image](https://user-images.githubusercontent.com/22032251/41140876-177b3e6c-6af0-11e8-9c1f-45f437e83052.png)

**Comment by diddipoeler** [comment]
Sieht schon gut aus 👍

**Comment by diddipoeler** [comment]
Gibt es das schon als Quellcode ?

**Comment by tdittmann** [comment]
Ja, habe gerade einen PR aufgemacht. War in letzter Zeit oft wegen geschäftlich eingespannt oder im Urlaub :)

**Comment by diddipoeler** [comment]
So, nach einigen Änderungen läuft es jetzt.
Siehe Homepage 👍

**Comment by diddipoeler** [comment]
Bitte auch an die korrekte Sprachdatei denken !

**Comment by Loschdi** [comment]
Ist es ein großer Aufwand, dass dieses Modul auch Projektübergreifend funktioniert? Alle Saisons der Mannschaft X seit 200X...?

**Comment by diddipoeler** [comment]
Bitte einen neuen Issue eröffnen !
